src/browser.js

```diff
@@ -0,0 +1 @@
+export default globalThis.browser || globalThis.chrome;
```

Comment:
There is a better "official" [polyfill](https://github.com/mozilla/webextension-polyfill) but I couldn't use it and bundle it, so I used a simple way for now.